### DISCUSSION TITLE
Alignment problem

### DISCUSSION THREAD

**Discussion by tpnce** [alicevision/Meshroom#2339] [Show and tell]
Hi everyone, 

I am a beginner on Meshroom and I am french, so please apologize if I make some mistakes
I encounter problems when I try to align my photos with photogrammetry pipeline. I think my images are pretty good, but it doesn't align properly. 
 
I used a **turntable over a white background and I shot every 10° with a Nikon D850 and standard zoom at 85mm, 64 ISO. My lighting was flash. File format is TIFF 16 bits. I shot from different angles (see below).** 

![Uploading 04.jpg…]()
![01](https://github.com/alicevision/Meshroom/assets/163834947/4cd91b3d-8cae-4358-9162-eae6e158b7dc)
![02](https://github.com/alicevision/Meshroom/assets/163834947/2288dec8-4428-47cf-9103-9dcbda8b7130)
![03](https://github.com/alicevision/Meshroom/assets/163834947/c50a7e7c-9024-4a5f-a4c1-3ec17369785c)

I also **made macrophotographies with another lens in order to improve the precision of the model** (see below).

![macro02](https://github.com/alicevision/Meshroom/assets/163834947/fbfacccf-55c1-490b-be76-ec8fd2bf1213)
![macro03](https://github.com/alicevision/Meshroom/assets/163834947/c3e56266-b377-47af-98c2-0f547217fc3a)
![macro01](https://github.com/alicevision/Meshroom/assets/163834947/7a1ccbcc-bc8f-4326-9402-25ebba7c0231)

I tried different combinations between the different groups of photos (only the images with 85mm, then all the images macro + 85mm, etc.) but it doesn't align. For example, this is the kind of alignment it makes : 
![Capture d'écran 2024-03-18 124811](https://github.com/alicevision/Meshroom/assets/163834947/13fba2d8-9fd6-435a-820e-becaa6bb7af1)

I think the object is a bit tricky because of its repetitive pattern, but **it aligns well with another set of photographs with a D850, same lens, 800 ISO, file format JPEG and continuous light** (see below) 
![black01](https://github.com/alicevision/Meshroom/assets/163834947/82e77ea0-e47d-4f00-a2bf-3efd0bd8f883)
Any advice ? 

Thank

**Comment by Flashback1010** [comment]
Hi,

There are few steps you should be aware of.

You’re pictures looks nice for human eyes but may not be for Photogrammetry.
If you want to take a set of photos with a **turntable using a flash**, you may want try to build/make a **“void”** first (pure black background without graduation on the turntable).

You also have an option that can help in _FeatureMatching_ → _Minimal motion 2D_
Default at -1, you can change it.

[Meshroom turntable](https://meshroom-manual.readthedocs.io/en/latest/tutorials/turntable/turntable.html)

Some of your pictures seems a bit blurry at the back of the object, you’re F-stop value may be a bit low.
Try to get an ‘even’ shape photo from the front to the back of your object by increasing the F-stop value.
